app/views/application/_header_navbar.html.erb

```diff
@@ -1,30 +1,21 @@
 <div id="header-navbar" class="container navbar navbar-static-top" role="navigation">
-  <div class="d-block d-md-none d-lg-none col-1 navbar-dark">
-    <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#mobile-menu" aria-controls="mobile-menu" aria-expanded="false" aria-label="Toggle navigation" style="background-color: seagreen">
-      <span class="navbar-toggler-icon"></span>
-    </button>
-  </div>
-  <div class="col-5 col-md-5 col-lg-4 logo-container">
+  <div class="col-6 col-md-5 col-lg-4 logo-container">
     <%= link_to root_path, class: "header-name", id: "header-logo" do %>
       <%= image_tag('temple-logo-t-box.svg', class: "header-logo") %> <span>University Libraries</span>
     <% end %>
   </div>
 
   <div id="main-search" class="col-6 col-lg-5 offset-lg-3 d-none d-md-block d-lg-block text-right">
-    <form action="https://librarysearch.temple.edu/bento">
-      <input type="text" name="q" id="librarySearch" placeholder="Search everything" autocomplete="off" style="border: solid #1f9584 1px !important" />
-      <input type="submit" value = "">
-    </form>
-    <ul class="text-left">
-      <li><%= link_to "Books", "#" %></li>
-      <li><%= link_to "Articles", "#" %></li>
-      <li><%= link_to "Databases", "#" %></li>
-    </ul>
+    <%= render "main-search" %>
   </div>
-  <div id="main-search" class="col-3 d-block d-md-none d-lg-none text-right">
-    <form action="https://librarysearch.temple.edu/bento">
-      <input type="submit" value = "">
-    </form>
+  <div id="mobile-search-button" class="col-3 d-block d-md-none d-lg-none text-right">
+    <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#mobile-search" aria-controls="mobile-search" aria-expanded="false" aria-label="Toggle search form">
+    <span class="navbar-toggler-icon" onclick="myFunction(this)">
```

Comment:
Is `onclick` an accessibility concern? Don't keyboards nav hitting enter no respond to it?

Reply:
The ticket for the event listener will cover that, just need to add the enter keycode as well.